CHANGELOG.md

```diff
@@ -6,6 +6,16 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 
 ## Master
 
+##### Breaking
```

Comment:
I classified it as breaking because it will cause huge changes to the integration. Actually it shouldn't break anything beside podfile installation hooks, which could be affected by any change. But then I would rather say it's an enhancement. This PR is still labeled as defect, because it predates #4147. Since that was merged, everything what is left here to do was more a known feature limitation, I guess.